crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -93,7 +98,19 @@ pub fn execute_call_entry_point(
         ),
     };
 
-    runtime_state.cheatnet_state.trace_data.exit_nested_call();
+    let mut resources = UsedResources {
+        execution_resources: ExecutionResources {
+            vm_resources: &resources.vm_resources - &resources_before_call.vm_resources,
+            syscall_counter: resources.syscall_counter.clone(),
+        },
+        l2_to_l1_payloads_length: vec![],
+    };
+    resources.subtract_syscall_counter(&resources_before_call.syscall_counter);
+
+    runtime_state
+        .cheatnet_state
+        .trace_data
+        .exit_nested_call(resources);
```

Comment:
This is the core logic to this PR. `l2_to_l1_payloads_length` is empty because when I tried extracting it from `CallInfo` then if I do calls like
test_code -> A -> B
where both A and B emit a message I got
```console
thread 'tokio-runtime-worker' panicked at crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs:107:61:
called `Result::unwrap()` on an `Err` value: InvalidOrder { object: "L2-to-L1 message", order: 1, max_order: 1 }
```

Reply:
You mean in blockifier? Nope, each one has its own. Only resources include resources from inner calls

Reply:
Decided to change it - everything related to l1 will be done in #1592